src/components/EntryForm.js

```diff
@@ -23,114 +23,193 @@ export default function EntryForm({ onCreateCard }) {
+            name="watertemp"
+            type="number"
+            min={-5}
+            max={50}
+            onChange={handleOnChange}
+            placeholder="°C"
+          />
+        </Part>
+        <Part>
+          <label htmlFor="watercolor">Water Color</label>
+          <select
+            id="watercolor"
+            name="watercolor"
+            type="text"
+            onChange={handleOnChange}
+          >
+            <option value="cloudy">cloudy</option>
+            <option value="normal">normal</option>
+            <option value="clear">clear</option>
+          </select>
+        </Part>
+        <Part>
+          <label htmlFor="waterlevel">Water Level</label>
+          <select
+            id="waterlevel"
+            name="waterlevel"
+            type="text"
+            onChange={handleOnChange}
+          >
+            <option value="low">low</option>
+            <option value="normal">normal</option>
+            <option value="high">high</option>
+          </select>
+        </Part>
+      </Fieldset>
+      <small>weather</small>
+      <Fieldset>
+        <Part>
+          <label htmlFor="weather">Weather</label>
+          <Input
+            id="weather"
+            name="weather"
+            type="text"
+            maxLength={100}
+            onChange={handleOnChange}
+          />
+        </Part>
+        <Part>
+          <label htmlFor="temperature">Temperature</label>
+          <Input
+            id="temperature"
+            name="temperature"
+            type="number"
+            min={-50}
+            max={50}
+            maxLength={100}
+            onChange={handleOnChange}
+            placeholder="°C"
+          />
+        </Part>
+        <Part>
+          <label htmlFor="airpressure">Air pressure</label>
+          <Input
+            id="airpressure"
+            name="airpressure"
+            type="number"
+            min={850}
+            max={1150}
+            onChange={handleOnChange}
+            placeholder="850 - 1150"
+          />
+        </Part>
+        <Part>
+          <label htmlFor="moon">Moon phase</label>
+          <Input
+            id="moon"
+            name="moon"
+            type="text"
+            maxLength={100}
+            onChange={handleOnChange}
+          />
+        </Part>
+        <Part>
+          <label htmlFor="wind">Wind direction</label>
+          <Input
+            id="wind"
+            name="wind"
+            type="text"
+            maxLength={100}
+            onChange={handleOnChange}
+          />
+        </Part>
+        <Part>
+          <label htmlFor="windspeed">Wind speed</label>
+          <Input
+            id="windspeed"
+            name="windspeed"
+            type="number"
+            min={0}
+            onChange={handleOnChange}
+            placeholder="min. 0kmH"
+          />
+        </Part>
+      </Fieldset>
+      <SubmitButton text="Submit" isAccent={true} />
+    </Form>
```

Comment:
Really nice and detailed form with very good accessibility! 🥇

Reply:
Thanks, Julia